_posts/2021-10-06-september-rubygems-updates.md

```diff
@@ -0,0 +1,45 @@
+---
+title: September 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+# September 2021 Monthly Updates 
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in September.
+
+## RubyGems News
+
+This month in RubyGems, We released a new version of RubyGems [`3.2.28`](https://blog.rubygems.org/2021/09/23/3.2.28-released.html) and Bundler [`2.2.28`]().
+
+In addition to that, RubyGems saw several bug fixes and updates this month, some of which include the following:
+
+- worked on gem rebuild implementation - [#4913](https://github.com/rubygems/rubygems/pull/4913).
+- made some RubyGems rake tasks support `bsdtar`, allowing them to work on FreeBSD and macOS - [#4934](https://github.com/rubygems/rubygems/pull/4934).
+- worked on Bundler version locking RFC implementation - [#4076](https://github.com/rubygems/rubygems/pull/4076)
```

Comment:
Hei @lauragift21, september's work has been mainly rubygems 3.2.28 and bundler 2.2.28, but since we didn't include rubygems 3.2.27 and bundler 2.2.27 in the previous update, I would include those here as well. Let me do a small recap of the changes in each release.

> On the rubygems side, we released version [3.2.27](https://github.com/rubygems/rubygems/blob/f19363f4341ce2b6ac82a32978e2e6cf7019493b/CHANGELOG.md#3227--2021-09-02), fixing an issue when installing some gems from GitHub private gem servers, fixing some unredacted credentials in verbose mode, and also improved loading the library by using `require_relative` for internal requires. We also released [3.2.28](https://github.com/rubygems/rubygems/blob/8506ddac3cba150df99c2a1cf7bfe662dc62e646/CHANGELOG.md#3228--2021-09-22) fixing a regression introduced by the redaction fix in 3.2.27, adding support for the MINGW-UCRT platform, making sure to not load the `uri` gem unnecessarily, and relaxing gemspec validations to allow descriptions that include the "TODO" string. Still unreleased, but we also made progress on a `gem rebuild` command that will allow exactly reproducing exisiting package builds.

> On the bundler side, we released version [2.2.27](https://github.com/rubygems/rubygems/blob/6bcbd0b257c1a4b24cde10ce2911d4cdc31d8ef8/bundler/CHANGELOG.md#2227-september-2-2021) which fixes a couple of `bundle check`regressions and a couple of issues with plugins, and a syntax errors on the generated Github Actions configuration in new gems. It also optimizes some requires, and adds support for redacting credentials using the `x-oauth-basic` form. We also released [2.2.28](https://github.com/rubygems/rubygems/blob/cecb83bf071524014d862a0a4bfcc99f93299346/bundler/CHANGELOG.md#2228-september-22-2021), which makes sure `bundle remove` automatically regenerates the lockfile, deprecating the `--install` flag, and which also updates the gemspec generated on new gems to use `example.com` as the sample gem server instead of the potentially malicious `mygemserver.com`. Still to be released, but we also make progress on the Bundler version locking RFC implementation.